snapcraft/commands/remote.py

```diff
@@ -101,6 +101,11 @@ def fill_parser(self, parser: argparse.ArgumentParser) -> None:
             action="store_true",
             help="acknowledge that uploaded code will be publicly available.",
         )
+        parser.add_argument(
```

Comment:
we should probably fail if any of the build fails, I would start with that as a default

Reply:
Perhaps a flag named `--allow-failure` to opt out?

@sergiusens - should we also offer an envvar like `SNAPCRAFT_REMOTE_BUILD_ALLOW_FAILURE`?

Reply:
You should be able to do it just like this:
https://github.com/snapcore/snapcraft/blob/2e064759459924dbe163be07b081d902b61bf5e4/snapcraft/commands/lifecycle.py#L74

Reply:
Done!